extensions/interactions/base_test.py

```diff
@@ -740,6 +718,14 @@ def test_interactions_demo_exploration(self):
 
         missing_interaction_ids = (
             all_interaction_ids - observed_interaction_ids)
-        self.assertEqual(len(missing_interaction_ids), 0, msg=(
-            'Missing interaction IDs in demo exploration: %s' %
-            missing_interaction_ids))
+        if list(missing_interaction_ids) != ['RatioExpressionInput']:
+            # TODO(#10212):Ignoring the RatioExpressionInput for now as
```

Comment:
Add space after ``:`` here to be consistent, as well.

Reply:
done